obs-vault/documentation/katex-math-expressions-tutorial.md

```
@@ -0,0 +1,281 @@
+### 行列
+
+行列のような表形式の数式は `\begin{...}` と `\end{...}` で囲まれた「環境」を使って作成します。環境の中では、`&` で列を区切り、`\\` で行を改行します。
+
+行列を囲む括弧の種類によって、いくつかの環境が用意されています。
+
+| 環境名 | 括弧の種類 | コード例 | 表示例 |
+| :--- | :--- | :--- | :--- |
+| matrix | なし | `\begin{matrix} a & b \\ c & d \end{matrix}` | $\begin{matrix} a & b \\ c & d \end{matrix}$ |
+| pmatrix | 丸括弧 () | `\begin{pmatrix} a & b \\ c & d \end{pmatrix}` | $\begin{pmatrix} a & b \\ c & d \end{pmatrix}$ |
+| bmatrix | 角括弧 [] | `\begin{bmatrix} a & b \\ c & d \end{bmatrix}` | $\begin{bmatrix} a & b \\ c & d \end{bmatrix}$ |
+| vmatrix | 縦線 || (行列式) | `\begin{vmatrix} a & b \\ c & d \end{vmatrix}` | $\begin{vmatrix} a & b \\ c & d \end{vmatrix}$ |
```

markdownlint-cli2 (0.17.2)

Table column count
Expected: 4; Actual: 6; Too many cells, extra data will be missing

(MD056, table-column-count)